hawkular-command-gateway-war/src/main/java/org/hawkular/cmdgw/ws/server/FeedCommWebSocket.java

```diff
@@ -66,19 +63,17 @@
     @OnOpen
     public void feedSessionOpen(Session session, @PathParam("feedId") String feedId) {
         MsgLogger.LOG.infoFeedSessionOpened(feedId, session.getId());
-        boolean successfullyAddedSession = connectedFeeds.addSession(feedId, session);
```

Comment:
IIRC this used to fail (return false) if there was already a feed connected with that feed ID. We must ensure only a single WS connection is made per feed - if a feed ID is attemped to be connected multiple times, we need to fail.